src/utils/transaction.js

```diff
@@ -237,16 +248,6 @@ export default {
         return base58_1.default.encode(signature)
```

Comment:
Rename `description` to `attachment`. And change `desBytes` to `attachmentBytes`.